dist/api.classic.js

```diff
@@ -111,7 +113,11 @@
             executeFn(disconnectFn)();
         }.bind(this), raiseError(errorFn, "connect error"));
     };
+    Device.prototype.isConnected = function() {
+        return this.connected;
+    };
     Device.prototype.disconnect = function(errorFn) {
+        this.connected = false;
```

Comment:
Perhaps the connected state should be exposed by each adapter to keep it atomic? Is the property kept in sync with reality i.e. if the device powers off?